google_guest_agent/main.go

```diff
@@ -175,6 +176,11 @@ func runAgent(ctx context.Context) {
 
 	agentInit(ctx)
 
+	if cfg.Get().Unstable.CommandMonitorEnabled {
+		cmdsrv := command.Init(ctx)
```

Comment:
One last minor bit, change `command.Init()` to return nothing (maybe error only if that's the case) and add a `command.Close()`, the `command.Init()` would set the `cmdMonitor` pointer then `command.Close()` will check if `cmdMonitor` pointer is nil, if it's nil then do nothing, if not nil then we do call  `cmdMonitor.Close()` and set the pointer to nil. For other parts of the guest-agent using it we can just use the `command.Get()` function (that's already returning `cmdMonitor`.